core/bin/zksync_server/src/main.rs

```diff
@@ -165,6 +170,12 @@ fn main() -> anyhow::Result<()> {
         secrets,
     )?;
 
+    if opt.only_verify_config {
+        node.build(opt.components.0)?;
+        tracing::info!("Config verification is successful!");
+        return Ok(());
+    }
```

Comment:
Would it make sense to move it below `if opt.genesis` in a following manner?

```rust
let node = node.build(opt.components.0)?;

if opt.no_run {
  tracing::info!("Node composed successfully; exiting due to --no-run flag");
  return Ok(())
}

node.run(observability_guard)
```

Reply:
makes sense for having observability installed